e2e/addon_types_test.go

```diff
@@ -12,7 +12,7 @@
  * limitations under the License.
  */
 
-package v1alpha1
+package e2e
```

Comment:
why is this moved?

Reply:
API tests need refer API definition such as AddToScheme (BuilderScheme has been upgraded) to launch client, which is different from previous scheme. 
Previous API location introduced in loop.

Reply:
Per discussion, move them back to apis/addon directory

0ce90df6cde7f399781b319c16cfe1ce40cddfb6